src/js/module/main/main.js

```diff
@@ -3,6 +3,8 @@ import MainViewDialog from 'module/main/dialog';
 import Const from 'const';
 import Button from '../component/button';
 
+const IS_PLANTING_CLASS = 'is-planting';
```

Comment:
[Similar case](https://github.com/komitywa/plantingjs/pull/135/files#diff-9420eabb1646655a845e7ae6a5145f99R8) - basically classes with `is` prefix are shared between javascript and css. [Use of those classes in sass](https://github.com/komitywa/plantingjs/pull/135/files#diff-8c6488ad7989e5a0610b8358d90c5d22R14)